stroeer/fragment/stage/v1/stage_service.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+import "stroeer/page/section/v1/section_page.proto";
+
+package stroeer.fragment.stage.v1;
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/fragment/stage/v1;stage";
+option java_multiple_files = true;
+option java_package = "de.stroeer.fragment.stage.v1";
+
+service StageService {
+  /**
+    ############################################################################
+    # Description
+    ############################################################################
+    Get single stages by requesting them via agreed tokens, e.g. "schlagzeilen" or "meistgelesen".
+
+    ############################################################################
+    # Status/Error scenario's
+    ############################################################################
+
+    scenario: found
+    description: service responded without encountering exceptions
+    gRPC status: OK
+    gRPC error payload: none
+    HTTP status: OK
+    cacheable: yes
+
+    scenario: internal
+    description: internal error while loading data
+    gRPC status: INTERNAL
+    gRPC error payload: none
+    HTTP status: 500
+    cacheable: no
+
+    scenario: timeout
+    description: timeout while loading data
+    gRPC status: DEADLINE_EXCEEDED
+    gRPC error payload: none
+    HTTP status: 504
+    cacheable: no
+  **/
+  rpc GetStages (GetStagesRequest) returns (GetStagesResponse) {
+  }
+}
+
+// Message to request stages independent of SectionPages, for client side rendering or ESIs
+message GetStagesRequest {
+  // an id is a token which is agreed between frontend and backend, for example "schlagzeilen"
+  repeated string ids = 1;
```

Comment:
Did you consider enumerations here? Feels like this is a finite list of possible Ids which could be selected in a section tool.

Reply:
This may be a big enum and change quite often, as there are not only ld-lists in it, but also synthethical stuff like `most-read` and such.
We'd have to update the FE, BE and SectionTool every time something will be added :/

Reply:
Das was MaNa sagt.